eth/test/test_pagination.ts

```diff
@@ -32,14 +36,16 @@ describe("TreasureHuntCreator", () => {
     )) as TreasureHuntCreator__factory;
 
     totalPlayers = accounts.length;
-    questsRootCid = cidToBytes("QmUYWv6RaHHWkk5BMHJH4xKPEKNqAYKomeiTVobAMyxsbz");
+    questsRootCid = cidToBytes(
+      "QmUYWv6RaHHWkk5BMHJH4xKPEKNqAYKomeiTVobAMyxsbz"
+    );
   });
 
   async function deploy(
     solutions: string[],
     questsRootCid: Uint8Array
   ): Promise<TreasureHuntCreator> {
-    const thc = await thcFactory.deploy(solutions, questsRootCid);
+    const thc = await thcFactory.deploy(solutions, [], questsRootCid);
```

Comment:
shouldn't there be a test that checks that the leaderboard result contains the keys as expected?

Reply:
This test only checks the pagination is correct. The test you are talking about is implemented in the `integration.ts` test suite, specifically here https://github.com/social-dist0rtion-protocol/thc/blob/be6dc51f93ed0d88b6b6bc3ca1ad9fc70fece79d/eth/test/integration.ts#L279-L285